contrib/utilities/print_deprecated.py

```diff
@@ -85,7 +85,11 @@ def __init__(self, line):
         git_tag_output = decode_and_split(git_tag_output)
         relevant_tags = [tag for tag in git_tag_output if "rc" not in tag]
         if relevant_tags:
-            self.release = relevant_tags[0][1:] # skip the prepended 'v'
+            tag = relevant_tags[0]
+            if tag[0] == 'v' and tag[1].isnumeric():
```

Comment:
another option is to use `-l` in the command above, i.e. 
```
$git tag --contains 1f1c471914abe70db2dafe91178fb7e174ceffe7 -l 'v[0-9].[0-9].[0-9]'
```
That way you force git to look at tags which satisfy the regex. I don't think there is much need in any other tags, so i would do it this way.